### PR TITLE
add public accessor to get instance url

### PR DESCRIPTION
**Pull request by AhmadWaleed**
This pr adds a public accessor method to forrest client for accessing underlying authentication instance url.

Use Case:

I wanted to add link on my app which redirects user to some record page on salesforce

### CODE REVIEW

File: src/Omniphx/Forrest/Client.php [hunk 1]
```diff
@@ -674,6 +674,21 @@ public function getClient()
...
+    /**
+     * Accessor to get instance URL
+     *
+     * @param bool $withVersion
+     * @return string
+     */
+    public function instanceURL($withVersion = false)
+    {
+        if ($withVersion) {
+            return $this->getBaseUrl();
```
Comment (omniphx) [inline src/Omniphx/Forrest/Client.php:686]: how about just converting `getBaseUrl();` to public and remove the withVersion parameter?

**Comment by Shaws-WD** [comment]
Sam,

 

I’ve emailed all three

 

Rick may be especially interesting since I find on linkedIn that he used to be a software engineer with Jackson Labs

https://www.linkedin.com/in/rick-palazola-9685b95/

 

Best regards,

Steve

File: src/Omniphx/Forrest/Client.php (hunk 1 above)
Comment (AhmadWaleed) [reply src/Omniphx/Forrest/Client.php:686]: I would still prefer this instanceURL over getBaseUrl cus this give you option whether you want base url or with vesion in my use case i ony needed base url not with version.

File: src/Omniphx/Forrest/Client.php [hunk 2]
```diff
@@ -674,6 +674,21 @@ public function getClient()
         return $this->httpClient;
     }
 
+    /**
+     * Accessor to get instance URL
+     *
+     * @param bool $withVersion
+     * @return string
+     */
+    public function instanceURL($withVersion = false)
```
[the remark this answers is not in the document]
Comment (AhmadWaleed) [reply src/Omniphx/Forrest/Client.php]: done

File: src/Omniphx/Forrest/Client.php (hunk 1 above)
Comment (omniphx) [reply src/Omniphx/Forrest/Client.php:686]: I'm suggesting is we end up with 2 public methods: `getInstanceUrl()` and `getBaseURL()`. For instance, `getInstanceURL(true)` is not going to be intuitive to someone unfamiliar with the API

Comment (AhmadWaleed) [reply src/Omniphx/Forrest/Client.php]: Yeah make sense

**Comment by omniphx** [comment]
Thanks @AhmadWaleed !

**Pull request merged**